addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2532,127 +2532,6 @@ export class OdooEditor extends EventTarget {
             this.editable.classList.add('o_row_resize');
         }
     }
-    /**
-     * Resizes a table in the given direction, by "pulling" the border between
-     * the given targets (ordered left to right or top to bottom).
-     *
-     * @private
-     * @param {MouseEvent} ev
-     */
-    _resizeTable(ev, direction, target1, target2) {
-        ev.preventDefault();
-        const position = target1 ? (target2 ? 'middle' : 'last') : 'first';
-        let [item, neighbor] = [target1 || target2, target2];
-        const table = closestElement(item, 'table');
-        const [sizeProp, positionProp, clientPositionProp] = direction === 'col' ? ['width', 'x', 'clientX'] : ['height', 'y', 'clientY'];
-
-        // Preserve current sizes.
-        const tableRect = table.getBoundingClientRect();
-        table.style[sizeProp] = tableRect[sizeProp] + 'px';
-        const unsizedItemsSelector = `${direction === 'col' ? 'td' : 'tr'}:not([style*=${sizeProp}])`;
-        for (const unsizedItem of table.querySelectorAll(unsizedItemsSelector)) {
-            unsizedItem.style[sizeProp] = unsizedItem.getBoundingClientRect()[sizeProp] + 'px';
-        }
-
-        // TD widths should only be applied in the first row. Change targets and
-        // clean the rest.
-        if (direction === 'col') {
-            let hostCell = closestElement(table, 'td');
-            const hostCells = [];
-            while (hostCell) {
-                hostCells.push(hostCell);
-                hostCell = closestElement(hostCell.parentElement, 'td');
-            }
-            const nthColumn = getColumnIndex(item);
-            const firstRow = [...table.querySelector('tr').children];
-            [item, neighbor] = [firstRow[nthColumn], firstRow[nthColumn + 1]];
-            for (const td of hostCells) {
-                if (td !== item && td !== neighbor && closestElement(td, 'table') === table && getColumnIndex(td) !== 0) {
-                    td.style.removeProperty(sizeProp);
-                }
-            }
```

Comment:
@Zynton 
Why would we need to clean the style property of those columns ? Aren't they supposed to be clean as no code should set them?

And if we want to clean the style property of the columns that are not in the first row, why not clean all the style property of all columns of all rows (except the first) in the entire table rather than just the neighbors of `ev.target`.

Reply:
I don't remember. But looking at the code, we are precisely _not_ cleaning the style properties of the neighbors of `ev.target`. I see `hostCells` is all ancestor tds of the table (`closestElement(table, 'td')` above) so it seems we're just cleaning the ancestor tds if the table is nested and making sure not to touch the first row and its neighbor.